### PR TITLE
Display perf tests results

### PR DESCRIPTION
**Pull request by tisnik**
New dashboard can be seen here, now with perf. tests results, SLA threshold and SLA results as well:
https://fabric8-analytics.github.io/dashboard/dashboard.html

### CODE REVIEW

**Comment by tisnik** [comment]
[test this]

**Comment by tisnik** [comment]
[test this]

File: dashboard/src/perf_tests.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        for analyse in PerfTests.INPUT_FILES.keys():
+            self.statistic[analyse] = self.compute_statistic_for_analysis(self.results[analyse])
+
+    def get_results(self):
+        """Getter for the 'results' attribute."""
+        return self.results
+
+    def get_statistic(self):
+        """Getter for the 'statistic' attribute."""
+        return self.statistic
```
Comment (jpopelka) [inline dashboard/src/perf_tests.py]: It's OK, but we [prefer properties](https://stackoverflow.com/questions/2627002/whats-the-pythonic-way-to-use-getters-and-setters) in Python, i.e. something like:

```python

@property
def statistic(self):
    return self._statistic
```

File: dashboard/src/dashboard.py [hunk 2]
```diff
@@ -244,6 +246,14 @@ def main():
         delete_work_files(repository)
         update_overall_status(results, repository)
 
+    perfTests = PerfTests()
+    perfTests.read_results()
```
[the remark this answers is not in the document]
Comment (tisnik) [reply dashboard/src/dashboard.py]: these are generated by performance tests, ie. in another project

[the remark this answers is not in the document]
Comment (jpopelka) [reply dashboard/src/dashboard.py]: Then what about calling this class for example `PerfTestsResults` ?

**Pull request merged**